miso-web/src/main/webapp/scripts/hot_index.js

```diff
@@ -0,0 +1,66 @@
+HotTarget.index = {
+  getCreateUrl: function() {
+    return Urls.rest.indices.create;
+  },
+  getUpdateUrl: function(id) {
+    return Urls.rest.indices.update(id);
+  },
+  requestConfiguration: function(config, callback) {
+    callback(config)
+  },
+  fixUp: function(index, errorHandler) {
+    if (index.family.fakeSequence) {
+      index.sequence = index.name;
+    }
+  },
+  createColumns: function(config, create, data) {
+    return [{
+      header: 'Family',
+      data: 'familyName',
+      readOnly: true,
+      include: true,
+      unpack: function(index, flat, setCellMeta) {
+        flat.familyName = index.family ? index.family.name : config.indexFamily.name;
+      },
+      pack: function(index, flat, errorHandler) {
+        if (!index.family) {
+          index.family = config.indexFamily;
+        }
+      }
+    }, HotUtils.makeColumnForInt("Position", true, 'position', HotUtils.validator.integer(true, 1, 2)),
+        HotUtils.makeColumnForText('Name', true, 'name', {
+          validator: HotUtils.validator.requiredTextNoSpecialChars
+        }), HotUtils.makeColumnForText('Sequence', !config.indexFamily.fakeSequence, 'sequence', {
+          validator: HotUtils.validator.requiredTextNoSpecialChars
```

Comment:
When adding unique dual indices to a library, the code currently expects that the correct Index 2 will have the same name as the Index 1. Is there a (sensible) way we could add a constraint to the create/update indices HoT to have it check that each Index 2's name corresponds to an Index 1's name? If not, then a text box indicating this expectation would probably do.

Reply:
This would mean that index 1 always needs to be created or updated before index 2, which has more potential to be annoying than helpful. I'll add a column description and also detail that better in the user manual